src/Components/Text.tsx

```diff
@@ -39,10 +39,15 @@ const textStyleNameToCss = {
   secondary: fonts.secondary.style,
 }
 
-const RawText: React.SFC<TextProps> = props =>
-  <p className={props.className}>
-    {props.children}
-  </p>
+const RawText: React.SFC<TextProps> = (props: TextProps) => {
+  const newProps: TextProps = { ...props }
+  delete newProps.textSize
+  delete newProps.textStyle
+  delete newProps.align
+  delete newProps.color
```

Comment:
Rather than creating a dupe and then deleting, you can leave them out and only create a new object with the remainder:

```js
const { textSize, textStyle, align, color, ...remainderProps } = props
```